core/arch/x86/decode_table.c

```diff
@@ -1273,6 +1273,62 @@ const instr_info_t * const op_instr[] =
     /* OP_adcx          */   &prefix_extensions[143][2],
+
+    /* AVX-512 VEX encoded (scalar opmask instructions) */
+    /* OP_kmovw         */  &vex_W_extensions[74][0],
+    /* OP_kmovb         */  &vex_W_extensions[75][0],
+    /* OP_kmovq         */  &vex_W_extensions[74][1],
+    /* OP_kmovd         */  &vex_W_extensions[75][1],
+    /* OP_kandw         */  &vex_W_extensions[82][0],
+    /* OP_kandb         */  &vex_W_extensions[83][0],
+    /* OP_kandq         */  &vex_W_extensions[82][1],
+    /* OP_kandd         */  &vex_W_extensions[83][1],
+    /* OP_kandnw        */  &vex_W_extensions[84][0],
+    /* OP_kandnb        */  &vex_W_extensions[85][0],
+    /* OP_kandnq        */  &vex_W_extensions[84][1],
+    /* OP_kandnd        */  &vex_W_extensions[85][1],
+    /* OP_kunpckbw      */  &vex_W_extensions[87][0],
+    /* OP_kunpckwd      */  &vex_W_extensions[86][0],
+    /* OP_kunpckdq      */  &vex_W_extensions[86][1],
+    /* OP_knotw         */  &vex_W_extensions[88][0],
+    /* OP_knotb         */  &vex_W_extensions[89][0],
+    /* OP_knotq         */  &vex_W_extensions[88][1],
+    /* OP_knotd         */  &vex_W_extensions[89][1],
+    /* OP_korw          */  &vex_W_extensions[90][0],
+    /* OP_korb          */  &vex_W_extensions[91][0],
+    /* OP_korq          */  &vex_W_extensions[90][1],
+    /* OP_kord          */  &vex_W_extensions[91][1],
+    /* OP_kxnorw        */  &vex_W_extensions[92][0],
+    /* OP_kxnorb        */  &vex_W_extensions[93][0],
+    /* OP_kxnorq        */  &vex_W_extensions[92][1],
+    /* OP_kxnord        */  &vex_W_extensions[93][1],
+    /* OP_kxorw         */  &vex_W_extensions[94][0],
+    /* OP_kxorb         */  &vex_W_extensions[95][0],
+    /* OP_kxorq         */  &vex_W_extensions[94][1],
+    /* OP_kxord         */  &vex_W_extensions[95][1],
+    /* OP_kaddw         */  &vex_W_extensions[96][0],
+    /* OP_kaddb         */  &vex_W_extensions[97][0],
+    /* OP_kaddq         */  &vex_W_extensions[96][1],
+    /* OP_kaddd         */  &vex_W_extensions[97][1],
+    /* OP_kortestw      */  &vex_W_extensions[98][0],
+    /* OP_kortestb      */  &vex_W_extensions[99][0],
+    /* OP_kortestq      */  &vex_W_extensions[98][1],
+    /* OP_kortestd      */  &vex_W_extensions[99][1],
+    /* OP_kshiftlw      */  &vex_W_extensions[100][1],
+    /* OP_kshiftlb      */  &vex_W_extensions[100][0],
+    /* OP_kshiftlq      */  &vex_W_extensions[101][1],
+    /* OP_kshiftld      */  &vex_W_extensions[101][0],
+    /* OP_kshiftrw      */  &vex_W_extensions[102][1],
+    /* OP_kshiftrb      */  &vex_W_extensions[102][0],
+    /* OP_kshiftrq      */  &vex_W_extensions[103][1],
+    /* OP_kshiftrd      */  &vex_W_extensions[103][0],
+    /* OP_ktestw        */  &vex_W_extensions[104][0],
+    /* OP_ktestb        */  &vex_W_extensions[105][0],
+    /* OP_ktestq        */  &vex_W_extensions[104][1],
+    /* OP_ktestd        */  &vex_W_extensions[105][1],
+
+    /* AVX-512 EVEX encoded */
+    /* XXX i#1312: TODO. */
```

Comment:
XXX is an optional change or improvement which this is not: should be FIXME or if we add TODO to the style guide (sounds fine to me: feel free to edit the wiki page to add it) then TODO.

Reply:
Done.